packages/kbn-esql-validation-autocomplete/src/definitions/builtin.ts

```diff
@@ -393,24 +393,12 @@ const likeFunctions: FunctionDefinition[] = [
   //     defaultMessage: 'Case insensitive equality',
   //   }),
   // },
-  {
-    name: 'like',
-    description: i18n.translate('kbn-esql-validation-autocomplete.esql.definition.likeDoc', {
-      defaultMessage: 'Filter data based on string patterns',
-    }),
-  },
   { name: 'not_like', description: '' },
-  {
-    name: 'rlike',
-    description: i18n.translate('kbn-esql-validation-autocomplete.esql.definition.rlikeDoc', {
-      defaultMessage: 'Filter data based on string regular expressions',
-    }),
-  },
   { name: 'not_rlike', description: '' },
 ].map(({ name, description }) => {
```

Comment:
@stratoula Should we keep the internationalized descriptions for these functions? If we import the descriptions from Elasticsearch following this PR, I don't think they will be internationalized.

Reply:
In see we keep them in the other generated files such as https://github.com/elastic/kibana/blob/main/packages/kbn-esql-validation-autocomplete/src/definitions/generated/aggregation_functions.ts